src/components/ProductList/index.jsx

```diff
@@ -6,12 +6,12 @@ function ProductList({ products }) {
     <>
       {products && (
         <div>
-          {(products.docs || []).map((doc) => (
+          {(products || []).map((doc) => (
```

Comment:
Minor comment, doc is not a significant variable name, maybe product would be better.
Along the same lines the field item could be renamed to name or itemName since it is more significant to what it is currently stored in that variable.

Reply:
Yes, you're right! 👍  we changed from `doc` to `product`. Thanks!

About, the field "item", it's a good suggestion!. Although, I think It will have some impact on other files (or another team).  But, I believe that a possible solution could be to talk about this good suggestion on Sunday and the team who has to work with the "Add Item" form (also ListView or other files) in the next issue could change it with more time. Well, it's only my opinion maybe there will be much better solutions.